airbyte-ci/connectors/connectors_qa/src/connectors_qa/checks/documentation.py

```diff
@@ -121,40 +192,194 @@ def check_main_header(self, connector: Connector, doc_lines: List[str]) -> List[
             )
         return errors
 
-    def check_sections(self, doc_lines: List[str]) -> List[str]:
+    def validate_links(self, docs_content) -> List[str]:
+        valid_status_codes = [200, 403, 401, 405, 429, 503]  # we skip 4xx due to needed access
+        links = re.findall("(https?://[^\s\`)]+)", docs_content)
```

Comment:
I don't know such cases. It can be possible only in case when doc file has link to itself, I guess. But workaround for it is adding this links after release in separate pr.